app/controllers/api/functions.php

```diff
@@ -446,17 +494,22 @@
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_TAG)
     ->param('functionId', '', new UID(), 'Function ID.')
-    ->param('command', '', new Text('1028'), 'Code execution command.')
+    ->param('entrypoint', '', new Text('1028'), 'Entrypoint File.')
     ->param('code', [], new File(), 'Gzip file with your code package. When used with the Appwrite CLI, pass the path to your code directory, and the CLI will automatically package your code. Use a path that is within the current directory.', false)
+    ->param('automaticDeploy', false, new Boolean(true), 'Automatically deploy the function when it is finished building.', false)
```

Comment:
```suggestion
    ->param('deploy', false, new Boolean(true), 'Automatically deploy the function when it is finished building.', false)
```

Seems more consistent